src/pluginmanager.cpp

```diff
@@ -114,46 +117,16 @@ PluginManager::eventHook(DmtcpEvent_t event, DmtcpEventData_t *data)
 
   case DMTCP_EVENT_REAL_TO_VIRTUAL_PATH:
 
-    for (int i = pluginManager->pluginInfos.size() - 1; i >= 0; i--) {
-      if (pluginManager->pluginInfos[i]->event_hook) {
-        pluginManager->pluginInfos[i]->event_hook(event, data);
-      }
-    }
-    break;
-
-  // Process ckpt barriers.
-  case DMTCP_EVENT_PRESUSPEND:
-    for (size_t i = 0; i < pluginManager->pluginInfos.size(); i++) {
-      if (pluginManager->pluginInfos[i]->event_hook) {
-        pluginManager->pluginInfos[i]->event_hook(event, data);
-      }
-    }
-    break;
-
-  case DMTCP_EVENT_PRECHECKPOINT:
-    for (size_t i = 0; i < pluginManager->pluginInfos.size(); i++) {
-      if (pluginManager->pluginInfos[i]->event_hook) {
-        pluginManager->pluginInfos[i]->event_hook(event, data);
-      }
-    }
-    break;
-
-  // Process resume/restart barriers in reverse-order.
   case DMTCP_EVENT_RESUME:
-    for (int i = pluginManager->pluginInfos.size() - 1; i >= 0; i--) {
-      if (pluginManager->pluginInfos[i]->event_hook) {
-        pluginManager->pluginInfos[i]->event_hook(event, data);
-      }
-    }
-  break;
-
   case DMTCP_EVENT_RESTART:
+  case DMTCP_EVENT_THREAD_RESUME:
+
```

Comment:
Here, we added a blank line after the case statements.  But in the PRESUSPEND case states, we do not have a blank line.  For consistency, we should either have a blank line in both situations, or no blank line in both situations.